### ISSUE TITLE
IndexSizeError on attempt add link to image

### ISSUE THREAD

**Issue by alex2389** [tinymce/tinymce#3469]
Firefox Developer Edition 53.0a2 (2017-02-02)
reproduce on this page: https://www.tinymce.com/docs/demo/full-featured/
1 select image
2 click insert/edit link
3 paste any url
4 click ok
editor crashes with error in console:

21:29:43.641 IndexSizeError: Index or size is negative or greater than the allowed amount 1 tinymce.min.js:6
	setRng https://cdn.tinymce.com/4/tinymce.min.js:6:20946
	G/</< https://cdn.tinymce.com/4/tinymce.min.js:11:19358
	setEditorTimeout/< https://cdn.tinymce.com/4/tinymce.min.js:2:9374

**Comment by NavigateCMS** [comment]
We've just updated to Firefox 53 and the error described happens on all our instances of TinyMCE 4.5.3.
@spocke Will you do something to solve this bug in the editor source code?

**Comment by brianlenz** [comment]
It looks to me like this issue is fixed in 4.5.7. I suspect that this fix from 4.5.4 did it?

>FIXED bug where setBaseAndExtend would throw exceptions on Chrome 58 when selecting images.

(Note that the actual method is `setBaseAndExtent`, not `setBaseAndExtend`.)

We saw the index size error reported here on Firefox 53 prior to 4.5.4. We also experienced an error with `setBaseAndExtent` in Chrome 58 (which also had to do with selecting images). After upgrading to 4.5.7, both issues appear to be resolved in Firefox 53 and Chrome 58. So perhaps they were related?
